app/_how-tos/kic-observability-promethus-grafana.md

```diff
@@ -0,0 +1,170 @@
+
+prereqs:
+  kubernetes:
+    gateway_api: true
+    prometheus: true
+
+cleanup:
+  inline:
+    - title: Uninstall KIC from your cluster
+      include_content: cleanup/products/kic
+      icon_url: /assets/icons/kubernetes.svg
+---
+
+## How it works
+
+{{ site.base_gateway }} provides a Prometheus plugin that exports service and route metrics automatically. 
+
+The Prometheus stack scrapes metrics from deployments that match the labels defined within a `servicemonitor` resource. The {{ site.kic_product_name }} Helm chart can automatically label your deployments and create a `servicemonitor` instance to enable Prometheus metrics scraping.
+
+## Enable Prometheus
+
+{{ site.base_gateway }} does not expose Prometheus metrics by default. To enable the metrics, create a `prometheus` plugin instance:
+
+{% entity_example %}
+type: plugin
+data:
+  name: prometheus
+  config:
+    status_code_metrics: true
+    bandwidth_metrics: true
+    upstream_health_metrics: true
+    latency_metrics: true
+    per_consumer: false
+{% endentity_example %}
+
+## Deploy sample services
+
+This how-to will deploy multiple services to your Kubernetes cluster to simulate a production environment.
+
+Deploy the services and create routing resources:
+
+```bash
+kubectl apply -f {{ site.links.web }}/manifests/kic/multiple-services.yaml -n kong
+```
+
+{% include /k8s/httproute.md release=page.release name='sample-routes' path='/billing,/comments,/invoice' service='billing,comments,invoice' port='80,80,80' skip_host=true %}
+
+## Generate traffic
+
+Once the service and routes are deployed, it's time to generate some fake traffic. Open a new terminal and run the following command:
+
+```bash
+while true;
+do
+  curl $PROXY_IP/billing/status/200
+  curl $PROXY_IP/billing/status/501
+  curl $PROXY_IP/invoice/status/201
+  curl $PROXY_IP/invoice/status/404
+  curl $PROXY_IP/comments/status/200
+  curl $PROXY_IP/comments/status/200
+  sleep 0.01
+done
+```
+
+## Access Grafana
+
+Grafana is an observability tool that can be used to observe Prometheus metrics over time. To access Grafana you will need to `port-forward` the services:
+
+```bash
+kubectl -n monitoring port-forward services/prometheus-operated 9090 &
+kubectl -n monitoring port-forward services/promstack-grafana 3000:80 &
+```
+
+You will also need to get the password for the admin user.
+
+Execute the following to read the password and take note of it:
+
+```bash
+kubectl get secret --namespace monitoring promstack-grafana -o jsonpath="{.data.admin-password}" | base64 --decode ; echo
+```
+
+Now, browse to [http://localhost:3000](http://localhost:3000) and use the username `admin` and the password that you made a note of.
+
+Once logged in, you will see a `Kong (official)` dashboard in the bottom left. Click on this link.
+
+## Metrics collected
```

Comment:
I'd move this whole section to https://deploy-preview-1238--kongdeveloper.netlify.app/kubernetes-ingress-controller/observability/prometheus/ since this is more reference content and not opinionated. Then do a "Next steps" on this how to with a link to it.